core/deployment/src/main/java/io/quarkus/deployment/steps/ClassTransformingBuildStep.java

```diff
@@ -154,6 +157,17 @@ public byte[] apply(String className, byte[] originalBytes) {
                     } else {
                         return originalBytes;
                     }
+                } catch (Throwable e) {
+                    if (continueOnFailure) {
+                        if (log.isDebugEnabled()) {
+                            log.errorf(e, "Failed to transform %s", className);
```

Comment:
`log.debugf`?

Reply:
Basically it's the difference between logging the huge stack trace or not. I could add a debug and an error but it is better to just have the single message.